doc/source/cn/guide/set-up-env.md

```diff
@@ -28,26 +28,26 @@ brew install node
 
 ```bash
 $ node -v
-v6.3.1
+v6.11.3
 $ npm -v
-3.10.3
+3.10.10
 ```
 
 通常，安装了 Node.js 环境，npm 包管理工具也随之安装了。因此，直接使用 npm 来安装 weex-toolkit。
 
 > npm 是一个 JavaScript 包管理工具，它可以让开发者轻松共享和重用代码。Weex 很多依赖来自社区，同样，Weex 也将很多工具发布到社区方便开发者使用。
 
-**注意: ** weex-toolkit 在 1.0.1 之后才支持初始化 Vue 项目，使用前请确认版本是否正确。
+**注意: ** 在`weex-toolkit`1.0.8版本后添加了npm5规范的`npm-shrinkwrap.json`用于锁定包依赖，故npm版本<5的用户需要通过`npm i npm@latest -g`更新一下npm的版本，使用前请确认版本是否正确。
```

Comment:
The `npm i npm@latest -g` command will update your npm package to the latest version. If command runs successful, your npm version will be greater than 5.0.